src/recipes/editor-plugin/index.md

```diff
@@ -0,0 +1,54 @@
+<!--
+  ~ Copyright (c) godot-rust; Bromeon and contributors.
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+# Editor plugins
+
+
+Using `EditorPlugin` types is very similar to the process used when [writing plugins in GDScript][gd-plugins].
+Unlike GDScript plugins, godot-rust plugins are registered automatically and cannot be enabled/disabled in the
+Project Settings plugins pane.
+
+Plugins written in GDScript are automatically disabled if they have a code error, but because Rust is a compiled language,
+you cannot introduce compile-time errors.
+
```

Comment:
Maybe a better way to say this is that errors that that would cause Gdscript-based plugins to be disabled are caught at compile time

Reply:
Maybe such:
> Plugins written in GDScript are disabled if an error occurs during compilation. Since Rust is a compiled language, your plugin will be registered with the engine immediately, and you won't see it in the list of plugins. An error at runtime or connection can lead to editor crash.

Reply:
> your plugin will be registered with the engine immediately

Immediately when? On load/reload of the dynamic library?

> , and you won't see it in the list of plugins.

I think this misses the connection to the error, e.g. "won't see it ... if an error occurs".

> An error at runtime or connection can lead to editor crash.

What do you mean with "connection" here?

Reply:
ok, you can change it later as you wish, this is not mine text :)